movie/src/main/java/com/example/presentation/ui/util/ViewAdapters.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.presentation.ui.util
+
+import android.view.View
+import androidx.core.view.isInvisible
+import androidx.core.view.isVisible
+import androidx.databinding.BindingAdapter
+
+@BindingAdapter("isVisible", "setInvisibleOnFalse", requireAll = false)
+fun View.setIsVisible(viewIsVisible: Boolean, setInvisibleOnFalse: Boolean = false) {
+    if (isVisible != viewIsVisible) {
+        if (setInvisibleOnFalse) {
+            isInvisible = viewIsVisible.not()
+        } else {
+            isVisible = viewIsVisible
+        }
+    }
+}
+@BindingAdapter("onClick")
+fun View.setOnClickLambda(onClick: () -> Unit) {
+    setOnClickListener { onClick() }
+}
```

Comment:
A few helper databinding adapters. These functions can be called in your xml layout to set values on the Views. The databinding library supplies a ton of databinding adapters just like these (for things like `android:text`, `android:backgroundColor`, etc, etc. Here is the full list

https://android.googlesource.com/platform/frameworks/data-binding/+/refs/heads/master/extensions/baseAdapters/src/main/java/android/databinding/adapters